TcpServer/ResponseServerAsync.cs

```diff
@@ -109,14 +108,9 @@ private void ServeClient(TcpClient client)
             _connectedUsers.Remove(username);
         }
 
-        private void TransmissionCallback(IAsyncResult _asyncResult)
-        {
-            _logger("Client disconnected");
-        }
-
-        private void Send(NetworkStream stream, string message)
+        private Task Send(NetworkStream stream, string message)
```

Comment:
Czemu tu nie trzeba już async?

Reply:
`async` trzeba użyć tyko po to żeby użyć `await`. W tym wypadku można Funkcję `Send` i tak trzeba awaitować (bo return type to `Task`) więc nie ma sensu robić tego dwa razy (wewnątrz `Send` i przy wywołaniu `Send`)

Reply:
`await` powinno się używać tylko w przypadku gdy potrzebujemy zwróconej wartości lub chemy mieć pewność, że dany Task się skończył. Funkcja Send nie potrzebuje żadnej z tych rzeczy.

Reply:
ok, czaje